superchain/configs/mainnet/superlumio.yaml

```diff
@@ -16,3 +16,5 @@ genesis:
     hash: "0x01711dc53a82967a34f4b38f7dff3ae8fa760882bb1e2ace50bb01e9ff420a9b"
     number: 0
   l2_time: 1708984631
+
+ecotone_time: 1710374401
```

Comment:
Good call out. If you check the full genesis.json file [here](https://api.conduit.xyz/file/v1/optimism/genesis/pontem-mainnet-0), you'll see that actually canyone was not activated at genesis on this chain (no shanghai time), and our genesis hash validation check confirms that. I can double check with them, our tooling was not scraping the hardfork times properly when the chain was added, so we need to manually put the correct times in now.

Reply:
I think it doesn't make a difference to the genesis hash check, which is our immediate blocker, since any time after genesis is equivalent with respect to that hash. But probably we don't want to omit those upgrades altogether, might be safest to set them to the same as the `ecotone_time` until we can get the exact time from the conduit folks.

Reply:
Note that in general, we don't want to do or allow activation of forks at the same block, if they don't activate at genesis, but at least be spaced one block apart.